thrift_tools/thrift_struct.py

```diff
@@ -232,8 +231,8 @@ def __eq__(self, other):
     def __ne__(self, other):
         return not self.__eq__(other)
 
-    def __str__(self):
-        return 'field_type=%s, field_id=%s, value=%s' % (
+    def __repr__(self):
+        return 'ThriftField{field_type=%s, field_id=%s, value=%s}' % (
```

Comment:
I usually write these using `(` and `)` so that they mimic the class's initialization call.